Dockerfile

```diff
@@ -1,9 +1,19 @@
-FROM daschswiss/nginx-server:1.1.3
+FROM nginx:1
 
 LABEL maintainer="support@dasch.swiss"
 
-RUN rm -rf /public/*
+# set default internal port for the server
+# can be overriden at runsite
+ENV NGINX_PORT 4200
+EXPOSE ${NGINX_PORT}
 
-COPY apps/dsp-app /public
+# Copy nginx configuration
+COPY ./nginx/default.conf.template /etc/nginx/templates/
+# COPY ./nginx/nginx.conf /etc/nginx/
```

Comment:
Is this commented out line necessary?

Reply:
yes, good catch

Reply:
oh man, now I need to try it out again on the DEV server first.